sklearn/ensemble/_hist_gradient_boosting/gradient_boosting.py

```diff
@@ -262,9 +263,12 @@ def fit(self, X, y, sample_weight=None):
         # there's no way to tell the scorer that it needs to predict binned
         # data.
         self._in_fit = True
+        n_threads = _openmp_effective_n_threads()
```

Comment:
Maybe we could add an explicit inline comment to explain that we use this help to take cgroups CPU quotas into account when determining the maximum number of threads to use.